cfgov/unprocessed/css/main.less

```diff
@@ -134,8 +134,6 @@
 @import (less) "pages/event.less";
 @import (less) "pages/external-site.less";
 @import (less) "pages/error.less";
-@import (less) "pages/press-resources.less";
-// @import (less) "pages/car-buying-guide.less";
```

Comment:
To keep this in scope you could keep this line in this PR (or add a TODO) and then strip it out with the associated Less file in another PR (can the whole car-buying guide go yet??)

Reply:
That file already doesn't exist. I removed the comment because it doesn't refer to anything.